bazel/tools/java/com/code_intelligence/jazzer/tools/FuzzTargetTestWrapper.java

```diff
@@ -98,14 +101,29 @@ public static void main(String[] args) {
     List<String> command = new ArrayList<>();
     command.add(driverActualPath);
     command.add(String.format("-artifact_prefix=%s/", outputDir));
-    command.add(String.format("--reproducer_path=%s", outputDir));
-    command.add(String.format("--cp=%s",
-        hookJarActualPath == null
-            ? jarActualPath
-            : String.join(System.getProperty("path.separator"), jarActualPath, hookJarActualPath)));
     if (System.getenv("JAZZER_NO_EXPLICIT_SEED") == null) {
       command.add("-seed=2735196724");
     }
+    if (usesJavaLauncher) {
+      System.setProperty("jazzer.reproducer_path", outputDir);
+      // Forward all jazzer.* properties to the JVM used by the Java driver via the java_binary's
+      // --wrapper_script_flag argument.
```

Comment:
Could you please explain a little bit more where `--wrapper_script_flag` is coming from and that `driverActualPath` can point to the wrapper script, if `usesJavaLauncher`? Perhaps also a link to the bazel docs.

Reply:
Added, please take a look.